biz.aQute.resolve/src/biz/aQute/resolve/AbstractResolveContext.java

```diff
@@ -137,6 +137,7 @@ protected synchronized void init() {
 		List<Capability> result = findProviders0(requirement);
```

Comment:
I think it would be better to change `findProviders0` to not return null. As this error could crop up again when someone calls that method.